roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -173,6 +173,12 @@ spec:
             # Wait for the datastore.
             - name: WAIT_FOR_DATASTORE
               value: "true"
+{% endif %}
+{% if calico_network_backend is defined and calico_network_backend == 'vxlan' %}
+            - name: FELIX_VXLANNVI
```

Comment:
I do not find documentation about this variable in the docs: https://docs.projectcalico.org/reference/felix/configuration

Any ideas where I can find info on this?

Reply:
Nice catch @Miouge1 ... correct variable is `FELIX_VXLANVNI` and not `FELIX_VXLANNVI`

Reply:
Thank you @Miouge1! I've corrected the typo.